packages/gravity-ui-web/src/sass/05-components/03-organisms/_list-cards-basic.scss

```diff
@@ -10,6 +10,7 @@
   @media all and (min-width: gravy-breakpoint(medium)) {
     display: flex;
     flex-wrap: wrap;
+    margin: 0 -$grav-sp-l;
```

Comment:
So, since we _do_ want the negative horizontal margins, we can bend our rules here. However, let's _not_ set top and bottom margins. That way the automatic vertical spacing will still work.

I just tried and `margin: inherit -$grav-sp-l;` doesn't seem to work, so I'm afraid you'll need to ditch the shorthand form and do:
```
margin-left: -$grav-sp-l;
margin-right: -$grav-sp-l;
```